### ISSUE TITLE
Admin Console rendering wrongly (single page, instead of clickable tabs across top bar)

### ISSUE THREAD

**Issue by holta** [iiab/iiab#492]
This particular run occurred on a fully updated Raspbian Lite 2017-09-07 running [jvonau/iiab branch iiab-install4](https://github.com/jvonau/iiab/tree/iiab-install4).

http://box/admin appears as a single long page &mdash; instead of the usual 6 clickable tabs across top bar: 
```
[Control] [Configure] [Install Content] [Utilities] [Help] [About]
```
@jvonau mentions getting the following error trying to access the Admin Console thru a different URL (http://10.8.0.6/admin) on the very same machine:

> [Mon Nov 06 19:37:56.434840 2017] [core:error] [pid 31409] (13)Permission denied: [client 10.8.0.10:51556] AH00035: access to /common/js/Markdown.Sanitizer.js denied (filesystem path '/library/www') because search permissions are missing on a component of the path, referer: http://10.8.0.6/admin/

**Comment by holta** [comment]
Hopefully this has nothing to do with https://github.com/iiab/iiab-admin-console/pull/41 ?

**Comment by holta** [comment]
@jvonau has more detail:

> apache error log shows [Mon Nov 06 20:30:36.462838 2017] [core:error] [pid 10089] (13)Permission denied: [client 10.8.0.10:51948] AH00035: access to /common/js/Markdown.Sanitizer.js denied (filesystem path '/library/www') because search permissions are missing on a component of the path, referer: http://10.8.0.6/admin/

Permissions **(and ownership too)** are different on this newly-installed IIAB test RPi3:
```
root@box:~# ls -ld /library
drwxr-x--- 10 proxy proxy 4096 Nov  6 20:09 /library
```
As compared to all other RPi3's:
```
root@box:~# ls -ld /library
drwxr-xr-x 20 root root 4096 Oct 27 20:41 /library
```

**Comment by holta** [comment]
@jvonau confirms that the above permissions & ownership need to be fixed, and then Admin Console once again renders properly.

**Comment by holta** [comment]
Solved with https://github.com/jvonau/iiab/commit/651755f2ff1dbd07814c8ac6da1852fb19649fa7 which also solved #491 miraculously.
